Doppler.Jobs.Server/Startup.cs

```diff
@@ -24,13 +34,66 @@ public Startup(IConfiguration configuration)
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
+            services.AddSingleton(typeof(ILoggerAdapter<>), typeof(LoggerAdapter<>));
+
             services.AddHangfire(x =>
             {
                 x.UseSQLiteStorage();
             });
 
+            var httpClientPolicies = new HttpClientPoliciesSettings();
+            Configuration.GetSection("HttpClient:Client").Bind(httpClientPolicies);
+            services.AddSingleton(httpClientPolicies);
+
+            var handlerHttpClient = new HttpClientHandler
+            {
+                ClientCertificateOptions = ClientCertificateOption.Automatic,
+                SslProtocols = SslProtocols.Tls12
+            };
+
+            services.AddHttpClient(httpClientPolicies.ClientName, c => { })
+                .ConfigurePrimaryHttpMessageHandler(() => handlerHttpClient)
+                .SetHandlerLifetime(TimeSpan.FromMinutes(5))
+                .AddPolicyHandler(GetRetryPolicy(httpClientPolicies.Policies.RetryAttemps));
+
+            var dopplerCurrencySettings = new DopplerCurrencySettings
+            {
+                CurrencyCodeList = Configuration.GetSection("DopplerCurrencyService:CurrencyCode")
+                    .Get<List<string>>()
+            };
+            Configuration.GetSection("DopplerCurrencyService").Bind(dopplerCurrencySettings);
+            services.AddSingleton(dopplerCurrencySettings);
+
+            var jobsConfig = new TimeZoneJobConfigurations
+            {
+                TimeZoneJobs = $"{Configuration["TimeZoneJobs"]}"
```

Comment:
Weird syntax, why did you write it in this way?

Reply:
I thought that is necessary to convert string but I read the documentation `Configuration["TimeZoneJobs"]` returns a string, I'll change this sentence, Thanks

![image](https://user-images.githubusercontent.com/6796523/77541548-905ce680-6e83-11ea-83b6-3c96dbbc5799.png)

Reply:
Is it not the same? Am I missing something?

```suggestion
                TimeZoneJobs = Configuration["TimeZoneJobs"]
```

Reply:
Yes It's the same, done